ui/app/components/app/asset-list-item/asset-list-item.scss

```diff
@@ -1,9 +1,16 @@
 .asset-list-item {
+  &__token-button {
+    font-size: unset;
+    background: unset;
+    padding-inline-start: 0;
+
+    & h2 {
+      display: inline-block;
+    }
+  }
+
   &__token-value {
     padding: 0 5px 0 0;
-    overflow: hidden;
```

Comment:
q: why were these styles removed?

Reply:
I think this is the last thing I have that is blocking for me

Reply:
This was marked as resolved, but it wasn't answered.

<details>
<summary>Before, with long symbols</summary>
<img src="https://user-images.githubusercontent.com/4448075/97757098-10a69280-1aca-11eb-8bef-a8cac56b7ca5.png" />

</details>

<details>
<summary>After, with long symbols</summary>
<img src="https://user-images.githubusercontent.com/4448075/97757150-2ae07080-1aca-11eb-83c2-521d354af26f.png" />

</details>